CITATION.cff

```diff
@@ -21,4 +21,5 @@ authors:
     orcid: https://orcid.org/0000-0003-0670-7480
 title: "Aviary: Hybrid assembly and genome recovery from metagenomes with Aviary"
 version: 0.8.3
+doi: 10.5281/zenodo.10158087
```

Comment:
Should this be like this, per https://citation-file-format.github.io/
```
identifiers:
  - type: doi
    value: 10.5281/zenodo.10158087
```

Reply:
I'm basing the format from the github provided example: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

Reply:
Actually I think your way might be better - I don't think the DOI is picked up by GitHub in their autogen citation if is the way I suggested. @AroneyS can you adjust in CoerM and Galah please?

Reply:
Yeah you can check on the lorikeet and rosella repos, the doi is included in the citation created by github using my method

Reply:
@wwood I copied Aviary formatting for Galah. CoverM doesn't have a zenodo yet. Did you flick the switch?

Reply:
Oh we need the github release for CoverM